### PR TITLE
[#1454] Rename instances of container and inner-container to contain prefix butter

### PR DESCRIPTION
**Pull request by stanica**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/b3eed2abbe92c79/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @k88hudson received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/0f0e19a488c13cf/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/b3eed2abbe92c79/output.txt

Total script time: 0.53 mins

Server is running on http://pr461.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/0f0e19a488c13cf/output.txt

Total script time: 0.37 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/6708c6282b2c45b/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/6708c6282b2c45b/output.txt

Total script time: 0.34 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/543a92f2cfdca45/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/543a92f2cfdca45/output.txt

Total script time: 0.17 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/a60e21a17d78641/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/a60e21a17d78641/output.txt

Total script time: 0.20 mins

File: external/popcorn-js [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit da087cb2107af5ac0b22180cc80f9033cb7a8647
+Subproject commit e71840d72a9b8a7c00dc77588d475bc12cec6082
```
Comment (humphd) [inline external/popcorn-js]: This looks suspicious, was it intended?

File: src/core/views/media-view.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 define( [ "ui/page-element", "ui/logo-spinner", "util/lang", "ui/widget/textbox", "text!layouts/media-view.html" ],
   function( PageElement, LogoSpinner, LangUtils, TextboxWrapper, HTML_TEMPLATE ){
 
-  var DEFAULT_SUBTITLE = "Supports HTML5 video and YouTube",
+  var DEFAULT_SUBTITLE = "Supports HTML5 video, YouTube, and Vimeo",
```
Comment (humphd) [inline src/core/views/media-view.js:4]: This is wrong, you must have a bad rebase.

**Comment by jbuck** [comment]
Staged on storycamp: https://github.com/mozilla/butter/compare/4817dfd7a468...d0fa2b4df30f

**Pull request closed without merging**